stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/repository/RepositoryHelper.kt

```diff
@@ -94,6 +99,33 @@ internal class RepositoryHelper(
         configsRepository.clearCache()
     }
 
+    internal suspend fun updateChannelByDeletedDate(cid: String, deletedAt: Date) {
+        channels.setDeletedAt(cid, deletedAt)
+    }
+
+    internal suspend fun updateLastMessageForChannel(cid: String, lastMessage: Message) {
+        selectChannelWithoutMessages(cid)?.also { channel ->
+            val messageCreatedAt = checkNotNull(
+                lastMessage.createdAt
+                    ?: lastMessage.createdLocallyAt
+            ) { "created at cant be null, be sure to set message.createdAt" }
+
+            val oldLastMessage = channel.lastMessage
+            val updateNeeded = if (oldLastMessage != null) {
```

Comment:
you can use only boolean logic here, no need to the if/else.

Reply:
Yes, you're right but then the condition will look so messy, I'd like to keep it as is